setup.py

```diff
@@ -1,11 +1,33 @@
+# dependencies for python 2 or python 3
+import sys
+if sys.version < '3':
+    requires = ['rope (>= 0.9.4)']
+    install_requires = ['rope >= 0.9.4']
+else:
+    requires = ['rope_py3k (>= 0.9.4)']
```

Comment:
I don’t want to do sanction ``rope_py3k`` as a semi-official alternative. I hope we will make the real rope working with py3k soon.